vitest.setup.ts

```diff
@@ -15,6 +15,10 @@ vi.mock("@/utils/debounce", () => ({
 	debounceAsync: (promise) => promise,
 }));
 
+vi.mock("@/utils/interval", () => ({
+	interval: (callback: () => void) => callback(),
+}));
```

Comment:
I think adding has a few implications:
- This leaks implementation details of `useConfirmedTransaction`, when implementation of hook changes this will be provide no value, besides having this along HTTP call mock makes stuff even worse
- Makes tests hard to reason, if I had to work on the `SendTransfer` tests having this hidden configs/overrides would take time to discover
- Having `interval` over `setInterval` for the sake of tests doesn't make sense to me..